src/gameLogic/Engine.js

```diff
@@ -1,11 +1,20 @@
 import BoardState from "./BoardState"
 import Ticker from "./Ticker"
+import Player from "./Player"
+import Unit from "./Unit"
+import MovementPattern from "./MovementPattern"
+import ArtificialIntelligence from "./ArtificialIntelligence"
 
 class Engine {
 
   constructor() {
-    this.boardState = new BoardState()
     this.ticker = new Ticker(this.tick.bind(this))
+
+    this.human = new Player("rgb(0, 255, 0)", 0.0)
```

Comment:
I might want to change that? As long as we have just one unit type on each side, we can do with having something like sphere as our own unit and triangle as an enemy unit. 

But we are planning on having more unit types early on. It's a huge strain on working memory and thus poor UI to remember "my speheres correspond to enemy triangles, my starts correspond to their..."

We need **some** way to solve that. It could be done with colours or it could be done with something like "all enemy units have a vertical line through them"?